test/integration/statistics_announcement_test.rb

```diff
@@ -0,0 +1,35 @@
+require 'test_helper'
+
+class StatisticsAnnouncementTest < ActionDispatch::IntegrationTest
+
+  test "official statistics are rendered correctly" do
+    setup_and_visit_content_item('official_statistics')
+
+    assert_has_component_title(@content_item["title"])
+    assert page.has_text?(@content_item["description"])
+    assert_has_component_metadata_pair("Release date", "20 January 2016 9:30am (confirmed)")
+  end
+
+  test "national statistics are rendered correctly" do
+    setup_and_visit_content_item('national_statistics')
+
+    assert_has_component_title(@content_item["title"])
+    assert page.has_text?(@content_item["description"])
+    assert page.has_css?('.national-statistics-logo img')
+    assert_has_component_metadata_pair("Release date", "January 2016 (provisional)")
```

Comment:
It feels like this test and the one above have a slightly odd relationship between their names and the last assertion. Is it genuinely a difference between the formats, or is it because each example happens to demonstrate one of provisional vs confirmed?